build.rs

```diff
@@ -2,14 +2,10 @@ use std::process::Command;
 
 fn main() {
```

Comment:
We should add the following command
```
    Command::new("npm")
        .current_dir("frontend")
        .arg("i")
        .status()
        .unwrap();
```

Reply:
Done in https://github.com/oeb25/Cuely/commit/9e18189e35ed6ea7cc888d4b2c79658a853bc436